configs/core.js

```diff
@@ -94,7 +94,8 @@ module.exports = {
 		'no-irregular-whitespace': ['error', {
 			"skipStrings": true, // Default
 			"skipComments": true,
-			"skipTemplates": true
+			"skipTemplates": true,
+			"skipRegExps": true
```

Comment:
Without the "skipRegExps" flag you get an error when you create a RegExp where you're intentionally matching/capturing non-standard space characters.

Something I've run into a couple of times recently.

BTW, VSCode's `editor.unicodeHighlight.*` settings helpfully highlight/outline all non-standard spaces inside RegExps and strings, so this change should be perfectly safe.